app/helpers/error_items_helper.rb

```diff
@@ -2,8 +2,8 @@ module ErrorItemsHelper
   def error_items(field, error_items)
     if error_items && error_items.select { |item| item[:field] == field }.any?
       sanitize(error_items
-        .select { |key| key.to_s.match(field) }
-        .map { |error| error[:text] }.uniq
+        .select { |item| item[:field].to_s.match(field) }
+        .map { |item| item[:text] }.uniq
```

Comment:
This would be a good candidate for `filter_map`:
```ruby
.filter_map { |item| item[:text] if item[:field].to_s.match(field) }.uniq
```

Reply:
Nice, I've not seen that function before